src/id_generator.py

```diff
@@ -0,0 +1,18 @@
+class IDGenerator:
+    def __init__(self):
+        self.current_id = 0
+        self.used_ids = []
+
+    def assign_id(self):
+        self.current_id += 1
+        while self.current_id in self.used_ids:
+            self.current_id += 1
+        self.used_ids.append(self.current_id)
+        return self.current_id
+
+    def assign_rocket_ID(self):
```

Comment:
This is restrictingly specific. Why not let choose the user an Id they want to create? There we could check whether it is still avaliable. For the case of having no preference, one can still use the above method to get one